### ISSUE TITLE
Temperature In Status Bar

### ISSUE THREAD

**Issue by HonestAbe7** [davidtakac/prognoza#16]
Please consider having the temperature in the status bar so I can see how cold it is outside (for now); or hot.

**Comment by davidtakac** [comment]
Do you mean Android's status bar, like in a notification?

**Comment by HonestAbe7** [comment]
Yes. That way I don't need to open the app to see the current temperature. Although, if you do decide to implement this you might want to add an interval in the settings where the app and widget are refreshed. For example every 15 minutes or every 30 minutes or every hour; especially if you add other weather data sources as discussed in the other post. 😃

**Comment by davidtakac** [comment]
That's a nice suggestion! I think Google Weather does the same thing. I'll see what I can do, thanks. 

> every 15 minutes or every 30 minutes or every hour

I'm against exposing these kinds of settings to users. The app should take care to always display accurate and up-to-date information. Refresh intervals are implementation details that the user doesn't need to know about. This makes the app more simple to use and it puts us in control to manage the load on the weather provider's server.
